nodebb-theme-fdk/lib/client.js

```diff
@@ -13,16 +13,51 @@ $(document).ready(function() {
 		}
 	});
 
-	(function(history){
-    var pushState = history.pushState;
-    history.pushState = function(state) {
+	(function (history) {
+		var pushState = history.pushState;
+		history.pushState = function (state) {
 			if (arguments[0].url === "") {
 				document.getElementById("fdk-splash").style.display = "";
 			} else {
 				document.getElementById("fdk-splash").style.display = "none";
 			}
-      return pushState.apply(history, arguments);
-    };
-})(window.history);
+			return pushState.apply(history, arguments);
+		};
+	})(window.history);
+
+	function getEnvironment() {
```

Comment:
```suggestion
	const getEnvironment = () => {
```

Reply:
Jeg og @azaxarov  har bestemt oss for å bruke minst mulig arrow-functions her for å unngå kompatibilitetsissues.